src/js/hopscotch.js

```diff
@@ -756,11 +756,11 @@
 
       // Determine label for next button
       if(isLast){
-        nextBtnText = utils.getI18NString('doneBtn');
+        nextBtnText = (step.i18n && step.i18n["doneBtn"]) || utils.getI18NString('doneBtn');
```

Comment:
Please use dot syntax here instead of brackets... `step.i18n && step.i18n.doneBtn`.

*(+ other instances)*